tasks/install-with-venv.yml

```diff
@@ -15,7 +15,7 @@
 - name: Install certbot DNS provider
   pip:
     name:
-      - "certbot-{{ certbot_dns_provider }}"
+      - "certbot-dns-{{ certbot_dns_provider }}"
```

Comment:
@hexylena was this previously working for you without `dns-`?

Reply:
Uhhhhh, good question. We only have one machine that uses DNS and it probably never re-exercised this route.

Reply:
Ahh ok good to know I'm probably not breaking something then